### DISCUSSION TITLE
Vercel deployment API issue

### DISCUSSION THREAD

**Discussion by alinarub** [neuefische/questions#771] [Web] [answered]
Hi all,

We passed all Github Vercel tests after we pushed, but our quiz results don't show on the [Vercel page](https://dog-app-lemon.vercel.app/quiz-page). We already tried redeploying. Do we maybe have to whitelist our API in the Next config?

Ty!!

**Comment by doemser** [comment, marked as answer]
Have you set all the necessary environment variables for your vercel deployment?

If not, make sure to set them and redeploy after you updated them.
